packages/web/stores/root.ts

```diff
@@ -55,6 +55,27 @@ import {
 const IS_TESTNET = process.env.NEXT_PUBLIC_IS_TESTNET === "true";
 const assets = AssetLists.flatMap((list) => list.assets);
 
+const RawAssetLists = AssetLists.map((assetList) => {
+  const assets = assetList.assets.map((asset) => {
+    const ibcTrace = asset.traces.find((trace) => trace.type === "ibc");
+    let base = asset.base;
+
+    if (ibcTrace) {
+      base = ibcTrace.counterparty.base_denom;
+    }
+
+    return {
+      ...asset,
+      base,
+    };
```

Comment:
Btw, we can get the source denom using the [`getSourceDenomFromAssetList`](https://github.com/osmosis-labs/osmosis-frontend/blob/e6fb7743580039d926d1448df0070506f4f34c5b/packages/utils/src/asset-utils.ts#L18) function in @osmosis-labs/utils.

Reply:
Done